src/Apps/Order/Components/Collapse.tsx

```diff
@@ -99,6 +103,7 @@ export class Collapse extends React.Component<
         style={{
           transition: "height 0.3s ease",
           overflow: "hidden",
+          visibility: open ? "visible" : "hidden",
```

Comment:
I added this to update the accessibility of the elements. Even though they're visually hidden, they were accessible with the keyboard